website/src/pages/subscriptions/create.astro

```diff
@@ -18,150 +18,182 @@ const variantFilter = {
 ---
 
 <BaseLayout title='Create new subscription' omitHeaderNavigation='true'>
```

Comment:
While you're on it, could you maybe satify the typechecker here: `omitHeaderNavigation={true}`?